functions/ecom.config.js

```diff
@@ -268,6 +268,15 @@ const app = {
       },
       "hide": false
     },
+    "greater_box": {
+      "schema": {
+        "type": "boolean",
+        "default": false,
+        "title": "Habilitar cálculo considerando maior caixa",
+        "description": "Ao selecionado, será considerado a maior dimensão e somado os pesos"
```

Comment:
```suggestion
    "use_bigger_box": {
      "schema": {
        "type": "boolean",
        "default": false,
        "title": "Calcular considerando apenas uma caixa",
        "description": "Se selecionado as dimensões não serão somadas, considerando o maior valor de cada dimensão entre os itens"
```

_greater_ tem outro sentido, tipo _mais legal_ ou _melhor_ 😄 
_maior_ no sentido literal é _bigger_ mesmo...